### ISSUE TITLE
handleRegisterTags(): deal sanely with duplicated mfgID

### ISSUE THREAD

**Issue by jbrzusto** [jbrzusto/motusServer#359]
- append ".N" suffix to `mfgID` if there is already a tag with the same `mfgID` registered in the current
or previous dateBin.

**Comment by cjardine-bsc** [comment]
Im away from my computer at the momemt so I cant double check, but Im fairly sure we have some processes on our end that rely on this being a number (used for moving tags between projects or checking for duplicate tag registrations) if there is not a strong reason to change this from a number to a character it would probably be best to leave it as it is.

**Comment by denislepage** [comment]
I think .N stands for a number, so no changes to the concept.

**Comment by jbrzusto** [comment]
Yeah, just an issue to myself - I shouldn't be relying on users submitting all their tags for a season in
one batch, so need to be smarter about handling duplicated mfgIDs among multiple tag registration
jobs, even if users have failed to use the decimal suffix.
